apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -103,6 +140,22 @@ const ConfigScreen = () => {
                 to create an access token in the Vercel dashboard.
               </HelpText>
             </FormControl>
+            <Box style={{ width: '100%' }}>
+              <Flex fullWidth flexDirection="column">
+                <Text fontWeight="fontWeightDemiBold" marginRight="spacing2Xs">
+                  Status
+                </Text>
+                <Box>
+                  {appInstalled && parameters.vercelAccessToken && tokenValid ? (
+                    <Badge variant="positive">Valid access token</Badge>
+                  ) : tokenError ? (
+                    <Badge variant="negative">Invalid access token</Badge>
+                  ) : (
+                    <Badge variant="warning">Token not configured</Badge>
+                  )}
```

Comment:
nit: Could this maybe be abstracted into an if/else or switch statement? Just a person preference but I find many conditionals more readable that way!